bpm_projects_api/apis/project.py

```diff
@@ -101,3 +155,35 @@ def delete(self, uid):
     def put(self, uid):
         """Update a project given its identifier"""
         return dao.update(uid, ns.payload)
+
+
+# route is chosen because of the example above the "T odo" class on
+# https://flask-restplus.readthedocs.io/en/0.11.0/example.html and
+# https://www.rithmschool.com/courses/flask-fundamentals/routing-with-flask
+# "<url>/<<data_type>:<variable_name>>
```

Comment:
do we need these comments?

Reply:
Those were for me so that i find the source again, they can be removed